_pages/css.md

```diff
@@ -360,54 +360,26 @@ the difference will often be negligible once the output file has been gzipped.
 [placeholders]: http://thesassway.com/intermediate/understanding-placeholder-selectors
 
 # Linting
-The styleguide provides a method of linting [Sass] code to ensure it conforms
-to the rules in the styleguide. This linting tool will go through all Sass code
-and issue warnings wherever the code differs from the styleguide. We've created
-a specific [`.scss-lint.yml` file][scss-lint yaml] that's configured to work
-with the css coding styleguide. There are two ways to set up linting:
-
-* locally with ruby
-* locally with node
-
-## Locally with ruby
-1. Add the [`.scss-lint.yml` file][scss-lint yaml] to the base of your
-   repository.
-2. Install the [scss-lint] gem with `gem install scss_lint`
-3. Run scss-lint: `scss-lint app/assets/stylesheets/`
-
-## Locally with node (experimental!)
-1. Run `npm install --save-dev @18f/stylelint-rules` to download the package and save it to your package.json
-2. The package provides both a gulp task and a cli interface. Detailed usage instructions can be found in the [README](https://github.com/18F/stylelint-rules)
-
-## Shortcomings
-
-### scss-lint
-
-The scss-lint tool currently lacks the functionality to check these rules in
-the CSS coding styleguide:
-
-- Does not limit line width to 80 characters
-- Does not check for numeric calculations in parentheses
-- Does not sort properties in quite the order we want (defaults to
-  alphabetical)
-
-### stylelint
-
-This tool is still being evaluated, so not every rule in our current styleguide
-is supported by stylelint. scss-lint is purpose built for SCSS and is therefore
-a bit more feature rich. The following rules are currently not supported:
-
-- PropertySpelling
-- UnecessaryParentReference
-
-That being said, if you want to avoid a dependency on ruby in your project and
-still benefit from reliable SCSS linting, please test out this tool!
+Using a linter will ensure that [Sass] code conforms to some consistent rules. A linting tool like 
```

Comment:
Since the recommended standard configuration and usage only applies to CSS (not Sass), should we replace that here?

```suggestion
Using a linter will ensure that CSS code conforms to some consistent rules. A linting tool like 
```

Alternatively, we could also mention [`stylelint-config-recommended-scss`](stylelint-config-recommended-scss) as a standard default for Sass projects.